main.go

```diff
@@ -98,6 +99,9 @@ func command() func() {
 		command.BoolVar(&config.Verbose, "V", false, "Set verbosity on proxy output")
 
 		command.Parse(os.Args[2:])
+		if !strings.HasPrefix(config.Domain, ".") {
```

Comment:
Could we add a test for it? We already have some tests that cover returning no command it can be used as example

Reply:
I added the test in commit 9aa188e